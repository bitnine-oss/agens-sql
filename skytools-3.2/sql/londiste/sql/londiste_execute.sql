
set log_error_verbosity = 'terse';

select * from londiste.execute_start('branch_set', 'DDL-A.sql', 'drop all', false);
select * from londiste.execute_start('branch_set', 'DDL-A.sql', 'drop all', false);

select * from londiste.execute_finish('branch_set', 'DDL-A.sql');
select * from londiste.execute_finish('branch_set', 'DDL-A.sql');
select * from londiste.execute_finish('branch_set', 'DDL-XXX.sql');

select * from londiste.execute_start('branch_set', 'DDL-B.sql', 'drop all', true);
select * from londiste.execute_start('branch_set', 'DDL-B.sql', 'drop all', true);



select * from londiste.execute_start('aset', 'DDL-root.sql', 'drop all', true);
select * from londiste.execute_start('aset', 'DDL-root.sql', 'drop all', true);
select * from londiste.execute_finish('aset', 'DDL-root.sql');
select * from londiste.execute_finish('aset', 'DDL-root.sql');

