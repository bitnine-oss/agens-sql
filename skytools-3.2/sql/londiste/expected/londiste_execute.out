set log_error_verbosity = 'terse';
select * from londiste.execute_start('branch_set', 'DDL-A.sql', 'drop all', false);
 ret_code |       ret_note       
----------+----------------------
      200 | Executing: DDL-A.sql
(1 row)

select * from londiste.execute_start('branch_set', 'DDL-A.sql', 'drop all', false);
 ret_code |                    ret_note                    
----------+------------------------------------------------
      201 | EXECUTE: "DDL-A.sql" already applied, skipping
(1 row)

select * from londiste.execute_finish('branch_set', 'DDL-A.sql');
 ret_code |          ret_note           
----------+-----------------------------
      200 | Execute finished: DDL-A.sql
(1 row)

select * from londiste.execute_finish('branch_set', 'DDL-A.sql');
 ret_code |          ret_note           
----------+-----------------------------
      200 | Execute finished: DDL-A.sql
(1 row)

select * from londiste.execute_finish('branch_set', 'DDL-XXX.sql');
 ret_code |                 ret_note                  
----------+-------------------------------------------
      404 | execute_file called without execute_start
(1 row)

select * from londiste.execute_start('branch_set', 'DDL-B.sql', 'drop all', true);
 ret_code |             ret_note              
----------+-----------------------------------
      401 | Node is not root node: branch_set
(1 row)

select * from londiste.execute_start('branch_set', 'DDL-B.sql', 'drop all', true);
 ret_code |             ret_note              
----------+-----------------------------------
      401 | Node is not root node: branch_set
(1 row)

select * from londiste.execute_start('aset', 'DDL-root.sql', 'drop all', true);
 ret_code |        ret_note         
----------+-------------------------
      200 | Executing: DDL-root.sql
(1 row)

select * from londiste.execute_start('aset', 'DDL-root.sql', 'drop all', true);
 ret_code |                     ret_note                      
----------+---------------------------------------------------
      201 | EXECUTE: "DDL-root.sql" already applied, skipping
(1 row)

select * from londiste.execute_finish('aset', 'DDL-root.sql');
 ret_code |            ret_note            
----------+--------------------------------
      200 | Execute finished: DDL-root.sql
(1 row)

select * from londiste.execute_finish('aset', 'DDL-root.sql');
 ret_code |            ret_note            
----------+--------------------------------
      200 | Execute finished: DDL-root.sql
(1 row)

